brouter-routing-app/src/main/res/values-ar/strings.xml

```diff
@@ -13,4 +13,60 @@
+    <string name="main_action_1">Download Manager</string>
+    <string name="main_action_2">BRouter App</string>
+
+    <string name="cancel">Cancel</string>
+    <string name="i_know">I know</string>
+    <string name="close">Close</string>
+    <string name="exit">Exit</string>
+    <string name="ok">OK</string>
+    <string name="success">Success</string>
+    <string name="error">An Error occurred</string>
+
+    <string name="title_download">BRouter Download Manager</string>
+    <string name="summary_download">*** Attention:***
+        \n\nThe Download Manager is used to download routing-data
+        files which can be up to 170MB each. Do not start the Download Manager
+        on a cellular data connection without a data plan!
+        Download speed is restricted to 16 MBit/s.</string>
+    <string name="title_timeoutfree">Successfully prepared a timeout-free calculation</string>
+    <string name="summary_timeoutfree">You successfully repeated a calculation that previously run into a timeout
+        when started from your map-tool. If you repeat the same request from your
+        maptool, with the exact same destination point and a close-by starting point,
+        this request is guaranteed not to time out.</string>
+    <string name="title_sdcard">Enter SDCARD base dir:</string>
+    <string name="title_action">Select Action</string>
+
+    <string name="action_via_select">Check VIA Selection:</string>
+    <string name="action_nogo_select">Check NoGo Selection:</string>
+    <string name="action_servermode">Server-Mode</string>
+    <string name="action_info">Info</string>
+    <string name="action_calc_route">Calc Route</string>
+    <string name="action_profile_settings">Profile Settings</string>
+    <string name="action_share">Share GPX</string>
+    <string name="action_select_from">Select from</string>
+    <string name="action_select_to">Select to/via</string>
+
+    <string name="msg_no_profile">no profile data</string>
+    <string name="msg_no_used_profile">, no used profile</string>
+    <string name="msg_too_much_data">Too much data for download. Please reduce.</string>
+    <string name="msg_download_start">Download scheduled. Check internet connection if it doesn\'t start.</string>
+    <string name="msg_wpt_selection">current waypoint selection:\n</string>
+    <string name="msg_no_wpt_selection">Expecting waypoint selection\n</string>
+    <string name="msg_alternative">Alternative</string>
+    <string name="msg_status_result">version = BRouter-%1$s \n
+        mem = %2$s \n
+        distance = %3$s km\n
+        filtered ascend = %4$s m\n
+        plain ascend = %5$s m\n
+        estimated time = %6$s</string>
+    <string name="msg_read_wpt_error">Error reading waypoints</string>
+    <string name="msg_no_wpt">coordinate source does not contain any waypoints!</string>
+    <string name="msg_to_much_wpts">"coordinate source contains too much waypoints: %1$d (please use from/to/via names)</string>
```

Comment:
The `"` in `"coordinate`should not be needed?

Reply:
You are right